next/components/forms/simple-components/Button.tsx

```diff
@@ -218,7 +218,7 @@ const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, PolymorphicProp
         className={style}
         {...buttonProps}
       >
-        <div className="flex items-center">
+        <div className="justify-center flex items-center">
```

Comment:
If this is to fix the issue you can see below on "Pokracovat" button, it probably does not work (at least not across all browsers, screenshot from Safari):

<img width="655" alt="Screenshot 2023-02-10 at 10 02 56" src="https://user-images.githubusercontent.com/2327312/218049311-909c91b5-e18a-48f0-aeb9-d952a84af81d.png">

Reply:
@mpinter do you mean missing icon in Pokracovat button ? i ddnt know until now that there s bug like that, but i would say its not problem on my side. Its problem on button component and I do not think i should fix thing slike these that with amount of work on this task.

Reply:
I see, did not realise it's becasue of the icon, likely a black icon on black background, though it's bad padding. Yes, as you've said, that's outside of scope of this task.